css/teams/1/Dashboard.css

```diff
@@ -0,0 +1,88 @@
+@import url(./colors.css);
+
+.ut-top-v-navbar {
+    display: flex;
+    flex-direction: column;
+    background-color: var(--color-primary-700);
+    height: 600px;
+    width: 7%;
+    justify-content: space-between;
+}
+
+.ut-top-v-navbar .top-v-navbar-list {
+    padding: 0;
+    margin-right: 1rem;
+    margin-left: 1rem;
+}
+
+.ut-top-v-navbar .nav-item {
+    font-family: Inter;
+    font-size: 16px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 24px;
+    letter-spacing: 0em;
+    list-style-type: none;
+    color: var(--color-gray-25);
+    margin-bottom: 0.5rem;
+}
+
+.ut-top-v-navbar .nav-item:hover {
+    background-color: var(--color-primary-800);
+}
+
+.ut-top-v-navbar .nav-brand {
+    list-style-type: none;
+    margin-top: -2rem;
+    margin-bottom: -2rem;
+}
+
+.ut-top-v-navbar .nav-search {
+    list-style-type: none;
+    margin-bottom: 1rem;
+}
+
+.ut-top-v-navbar .nav-item a {
+    text-decoration: none;
+}
+
+.ut-top-v-navbar .nav-item-login {
+    display: block;
+    border: none;
+    background: none;
+    color: var(--color-gray-25);
+    margin: 10px 30px 10px 18px;
+    cursor: pointer;
```

Comment:
Why change the cursor type?
i mean it’s fancy but …

Reply:
Usually it's used to indicate you're hovering over a link, but I actually remove it in my last commit.